data/net/minecraft/server/commands/ExperienceCommand.mapping

```diff
@@ -21,3 +41,13 @@ CLASS net/minecraft/server/commands/ExperienceCommand
 			ARG 4 add
 			ARG 5 set
 			ARG 6 query
+		METHOD lambda$static$0 (Lnet/minecraft/server/level/ServerPlayer;Ljava/lang/Integer;)Z
+			ARG 0 player
+			ARG 1 xp
+		METHOD lambda$static$1 (Lnet/minecraft/server/level/ServerPlayer;)I
+			ARG 0 player
+		METHOD lambda$static$2 (Lnet/minecraft/server/level/ServerPlayer;Ljava/lang/Integer;)Z
+			ARG 0 player
+			ARG 1 xp
```

Comment:
```suggestion
			ARG 1 experienceLevels
```

Unless I'm mistaken, these two lambdas are referring to different values (as per `Type`) and this is a clearer name for each.